packages/vite-plugin/src/partials/meta.ts

```diff
@@ -47,11 +47,13 @@ export default meta;
         }
 
         timeStamps[source] = Date.now();
-        await fs.promises.writeFile(
-          source,
-          JSON.stringify(data, undefined, 2),
-          'utf8',
-        );
+        if (!process.env.META_NOWRITE) {
```

Comment:
I think we should make the name more descriptive maybe `DONT_WRITE_TO_META_FILES` or something like that